src/components/kytos/napp/ActionMenuItem.vue

```diff
@@ -6,7 +6,60 @@
 
 <script>
 import Vue from 'vue'
-import httpVueLoader from "./httpVueLoader.js"
+import { loadModule } from "vue3-sfc-loader"
+ 
+const options = {
+
+  moduleCache: {
+    vue: Vue
+  },
+
+  pathResolve({ refPath, relPath }, options) {
+
+    if (relPath === '.')
+      return refPath;
+
+    if (relPath[0] !== '.' && relPath[0] !== '/')
+      return relPath;
+
+    return String(new URL(relPath, refPath === undefined ? window.location : refPath));
+  },
+
+  handleModule: async function (type, getContentData, path, options) {
+
+    switch (type) {
+      case '.css':
+        options.addStyle(await getContentData(false));
+        return null;
+      case '.kytos':
+        console.log("Kytos extension detected. Switch extension to .vue");
```

Comment:
This console log will be useful. Let's not also forget to update the changelog and to also update the changelog about this breaking change (no longer supporting `.kytos` UI files, which you already mentioned in the description of PR but needs to be in the changelog a breaking change)